.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I will remove this in our master branch. We don't use a `.vscode/launch.json` file in this project